fundamentals/intro.tex

```diff
@@ -83,19 +84,20 @@ \subsection{Background}
 
 Nuclear fuel cycle simulators drive \gls{RDD} by calculating `metrics',
 quantitative measures of performance that can be compared among fuel cycle
-options. The feasibility of the technology development and deployment
-strategies which comprise a fuel cycle option, the dynamics of transitions
-between fuel cycles, and many other measures of performance can be expressed in
-terms of these metrics. For example, economic feasibility is often measured in
-\gls{LCOE}, requiring calculation of lifetime
-fuel costs and electricity generation, while environmental performance might be
-measured by spent fuel volume, toxicity, or mined uranium.  A meta-analysis of
-fuel cycle systems studies identified over two dozen unique quantitative
-metrics spanning economics and cost, environmental sustainability and waste
-management impacts, safety, security and nonproliferation, resource adequacy
-and utilization, among others. With few exceptions, these metrics are derived
-from mass balances calculated by a fuel cycle simulator.
-\cite{flicker_evaluation_2014}.
+options.  The feasibility of the technology development and deployment
+strategies which comprise a fuel cycle option, the operational features of
+nuclear energy systems, the dynamics of transitions between fuel cycles, and
+many other measures of performance can be expressed in terms of these metrics.
+For example, economic feasibility is often measured in \gls{LCOE}, requiring
+calculation of lifetime fuel costs and electricity generation, while
+environmental performance might be measured by spent fuel volume, toxicity, or
+mined uranium.  A meta-analysis of fuel cycle systems studies identified over
+two dozen unique quantitative metrics spanning economics and cost,
+environmental sustainability and waste management impacts, safety, security and
+nonproliferation, resource adequacy and utilization, among others
+\cite{flicker_evaluation_2014}. With few exceptions (e.g.
+\cite{poinssot_assessment_2014}), these metrics are derived from mass balances
```

Comment:
my reading of the reviewer's comment is that they would like a short listing of these, such as:
```
With few exceptions (e.g. water pollution and land use \cite{poinssot_assessment_2014})...
```